src/herder/test/HerderTests.cpp

```diff
@@ -1451,12 +1613,16 @@ TEST_CASE("SCP Driver", "[herder][acceptance]")
 {
     SECTION("protocol 10")
     {
-        testSCPDriver(10, 10, 10, false, false);
+        testSCPDriver(10, 10, 10, false, false, false);
```

Comment:
let's remove the behavior pre protocol 11 in a previous commit

Reply:
Done.